sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statements.scala

```diff
@@ -278,6 +278,11 @@ case class InsertIntoStatement(
  */
 case class UseStatement(isNamespaceSet: Boolean, nameParts: Seq[String]) extends LeafParsedStatement
 
+/**
+ * A SetCatalog statement, as parsed from SQL.
+ */
+case class SetCatalogStatement(nameParts: Option[String]) extends LeafParsedStatement
```

Comment:
Can't we just make this `nameParts: String` without `Option`?

Reply:
Thank you for your advice. Done